core/node/events/stream.go

```diff
@@ -776,6 +776,59 @@ func (s *Stream) addEventLocked(
 	return newSV, nil
 }
 
+// Backfill retrieves the stream content since the given cookie and sends it to the callback.
+func (s *Stream) Backfill(
```

Comment:
Instead of calling this `Backfill` can it be renamed to something like `UpdatesSinceCookie`? Backfill makes sense in the context of subscriptions but this is on the Stream type and I think renaming it would be more clear for users that don't see it in the context of a stream update subscription.